examples/acoustic_example.py

```diff
@@ -106,21 +106,26 @@ def damp_boundary(damp, h, nbpml):
             damp[:, :, -(i + 1)] += val
 
 nrec, _ = data.traces.shape
-m = DenseData("m", model.vp.shape, dtype)
+m = DenseData(name="m", shape=model.vp.shape, dtype=dtype)
 m.data[:] = model.vp**(-2)
-u = TimeData("u", m.shape, nt, time_order=t_order, save=False, dtype=m.dtype)
-v = TimeData("v", m.shape, nt, time_order=t_order, save=False, dtype=m.dtype)
+u = TimeData(name="u", shape=m.shape, time_dim=nt, time_order=t_order, save=False, dtype=m.dtype)
```

Comment:
I know what you mean, but since we "own" the time buffering, ie. we decide how many buffers, I would like the time buffering to be opaque to the outside code. The docstring should make it clear that we  mean space only, but we might want to expand on this a little?